config/data_scrubber.go

```diff
@@ -17,11 +19,19 @@ var (
 		"secret", "credentials", "stripetoken"}
 )
 
+const (
+	defaultCacheTTL = 100
```

Comment:
100 seems a little high. That's a cache life of 16 minutes. Can we make this lower? 25?